### DISCUSSION TITLE
Is there a way to opt out from stream management?

### DISCUSSION THREAD

**Discussion by fjwong** [xmppjs/xmpp.js#941] [Q&A] [answered]
Hello, we are trying to migrate our application to use xmpp.js at the moment.

Unfortunately, it appears stream management is not working properly on our XMPP server and we would like to disable it in the meantime until this is fixed server-side.

Is there some way to easily disable stream management in xmpp.js?

**Comment by sonnyp** [comment, marked as answer]
You can simply build your own client by copy pasting the content of 

https://github.com/xmppjs/xmpp.js/blob/main/packages/client/index.js or https://github.com/xmppjs/xmpp.js/blob/main/packages/client/browser.js and removing what you don't want/need.

**Reply by fjwong** [reply]
I see. I was suspecting that would be the case, but wanted to confirm if there was an easier way of doing so.

Thank you for the confirmation!
